diagnostics-app/src/app/telemetry/telemetry-dashboard/telemetry-dashboard.component.css

```diff
@@ -1,48 +1,15 @@
 /*
- * Copyright 2021 The Chromium Authors. All rights reserved.
```

Comment:
Don't change the copyright year. See [go/copyright](https://goto.google.com/copyright).

Reply:
Thanks for the reminder. I changed it following the rule.